picard/browser/filelookup.py

```diff
@@ -49,6 +50,16 @@
 
 class FileLookup(object):
 
+    RE_MB_ENTITY = re.compile(r"""
+        \b(?P<entity>area|artist|instrument|label|place|recording|release|release-group|series|track|url|work)?
+        \W*(?P<mbid>[a-f0-9]{8}(?:-[a-f0-9]{4}){3}-[a-f0-9]{12})
+    """, re.VERBOSE | re.IGNORECASE)
+
+    RE_MB_CDTOC = re.compile(r"""
+        \b(?P<entity>cdtoc)
+        \W*(?P<mbid>[a-z0-9-_.]{28})
```

Comment:
This is misleading, since this part of the expression doesn't actually match a mbid (nor cdtoc is an actual entity but ok).

I think we should just use `discid` and change the code below.

```python
        if m is None:
            m = self.RE_MB_CDTOC.search(string)
            if m is None:
                return False
...
        if entity != 'cdtoc':
            mbid = m.group('mbid').lower()
          
...
        elif entity == 'cdtoc':
            discid = m.group('discid')
            disc = Disc(id=discid)
            disc.lookup()
```

Reply:
We could also rename the variable to `id`

Reply:
I renamed the variable to ID. The URLs are in both cases exceptionally the same, that way we don't have to deal with different code paths for reading the IDs later on.